### PR TITLE
[Server42] Connection to Test lab

### PR DESCRIPTION
**Pull request by nikitamarchenko**
Add ability to test server42 related rules on local OVS Test lab

- [Documentaion](https://github.com/telstra/open-kilda/blob/23917dff1850f1201497aaba139785fa60c05d0a/docs/design/server42/server42-to-lab/README.md)
- PR:#4468

### CODE REVIEW

**Review by timofei-durakov — APPROVED** [review]

File: src-cpp/server42/Dockerfile [hunk 1]
```diff
@@ -12,4 +12,6 @@ WORKDIR /root/cmake-build-release/
 
 COPY ./docker_entrypoint.sh /root/cmake-build-release/
```
Comment (rtretyak) [inline src-cpp/server42/Dockerfile:13]: should be removed since `.sh` file does not exist anymore. This fails the image build atm

**Review by timofei-durakov — COMMENTED** [review]
is local_loop.sh still used? If yes, please add a small comment in it with scenario when and where it might be useful

File: src-cpp/server42/Dockerfile (hunk 1 above)
Comment (nikitamarchenko) [reply src-cpp/server42/Dockerfile:13]: done

**Comment by nikitamarchenko** [comment]
> is local_loop.sh still used? If yes, please add a small comment in it with scenario when and where it might be useful

tnx, done!

**Review by timofei-durakov — APPROVED** [review]

**Review by rtretyak — APPROVED** [review]

**Review by niksv — APPROVED** [review]

**Pull request merged**